example/ssd/dataset/iterator.py

```diff
@@ -185,6 +183,10 @@ def _data_augmentation(self, data, label):
         else:
             interp_methods = [cv2.INTER_LINEAR]
         interp_method = interp_methods[int(np.random.uniform(0, 1) * len(interp_methods))]
-        data = resize(data, self._data_shape, interp_method)
-        data = transform(data, self._mean_pixels)
+        data = mx.img.imresize(data, self._data_shape[0], self._data_shape[1], interp_method)
+        data = mx.nd.transpose(data, (2,0,1))
+        data = data.astype(float)
+        data[0] -= self._mean_pixels[0]
```

Comment:
mxnet.base.MXNetError: [21:02:23] src/operator/tensor/./../elemwise_op_common.h:54: Incompatible attr in node  at 1-th input: expected (3,224,224), got (3,1,1)